ai/client/book_client.py

```
@@ -0,0 +1,23 @@
+def get_story(book_id, page_number, authorization):
+    endpoint = f"/v1/book/{book_id}/story/{page_number}"
+    result = api.request("GET", endpoint, authorization)
+    return result.get("story_text")
+
+#모든 스토리 받아오는 함수 (임시)
+def get_all_story(book_id, authorization):
+    endpoint = f"/v1/book/{book_id}/story" #임시
+    try:
+        result = api.request("GET", endpoint, authorization)
+        return result.get("story_text")
+    except :
```

Ruff (0.8.2)

Do not use bare `except`

(E722)